e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js

```diff
@@ -129,6 +129,19 @@ describe('Insurance - Your business - Nature of your business page - As an Expor
         partials.errorSummaryListItems().should('have.length', 3);
       });
     });
+
+    describe(`when ${EMPLOYEES_UK} is correctly entered with a comma`, () => {
+      it(`should not display  ${EMPLOYEES_UK} validation errors`, () => {
```

Comment:
Shoudl we add a test for "it should render the submitted value without commas"?

Reply:
so it should render with commas unless going back to the page later, still need the test?

Reply:
For MVP we're not re-adding commas when revisiting the page